doc/ApiDocGeneration/Generate-Api-Docs.ps1

```diff
@@ -130,15 +131,15 @@ Copy-Item "${DocGenDir}/assets/docfx.json" -Destination "${DocOutDir}" -Recurse
 New-Item -Path "${DocOutDir}" -Name templates -ItemType directory
 Copy-Item "${DocGenDir}/templates/**" -Destination "${DocOutDir}/templates" -Recurse -Force
 
-Write-Verbose "Create Toc for Site Navigation"
-New-Item "${DocOutDir}/toc.yml" -Force
-Add-Content -Path "${DocOutDir}/toc.yml" -Value "- name: ${ArtifactName}`r`n  href: api/`r`n  homepage: api/index.md"
+# Write-Verbose "Create Toc for Site Navigation"
```

Comment:
Should we be undoing these commented out lines?

Reply:
The Toc under root originally is referenced by index.html, but it is not currently used anywhere. It could clean up as it doesn't provide any functionalities.